bindings/py/cpp_src/bindings/sdr/py_SDR.cpp

```diff
@@ -30,6 +30,7 @@
 namespace py = pybind11;
 
 using namespace nupic;
+using namespace nupic::sdr;
```

Comment:
please move these "using .." declarations under the `namespace nupic_ext {` , 
otherwise, the using namespaces are used globally

Reply:
Ok, I will review this and move them as appropriate.

In the case you've highlighted here: it will not leak into the global namespace because this is a source file which will not be included by anyone else?  For the header files I agree with you.

Reply:
> is a source file which will not be included by anyone else? For the header files I agree  

ok, my bad, this is a cpp. You may leave it be.